spec/models/user_spec.rb

```diff
@@ -15,4 +18,20 @@
     expect(user).to_not be_valid
     expect(user.errors[:password_confirmation]).to_not be_blank
   end
+
+  describe '.admin?' do
```

Comment:
These tests look like they should be on the branch that adds the admin functionality?